query-engine/connector-test-kit-rs/query-engine-tests/tests/queries/data_types/native/postgres.rs

```diff
@@ -0,0 +1,325 @@
+                time_tz    DateTime @test.Timetz(3)
+                time_tz_2  DateTime @test.Timetz(3)
+                ts         DateTime @test.Timestamp(3)
+                ts_2       DateTime @test.Timestamp(3)
+                ts_tz      DateTime @test.Timestamptz(3)
+                ts_tz_2    DateTime @test.Timestamptz(3)
+
+                parent Parent?
+            }"#
+        };
+
+        schema.to_owned()
+    }
+
+    #[connector_test(schema(schema_date))]
+    async fn dt_native(runner: Runner) -> TestResult<()> {
+        create_row(
+            &runner,
+            r#"{
+            id: 1,
+            child: { create: {
+                id: 1,
+                date: "2016-09-24T00:00:00.000Z"
+                date_2: "2016-09-24T00:00:00.000+03:00"
+                time: "1111-11-11T13:02:20.321Z"
+                time_2: "1111-11-11T13:02:20.321+03:00"
+                time_tz: "1111-11-11T13:02:20.321Z"
+                time_tz_2: "1111-11-11T13:02:20.321+03:00"
+                ts: "2016-09-24T14:01:30.213Z"
+                ts_2: "2016-09-24T14:01:30.213+03:00"
+                ts_tz: "2016-09-24T14:01:30.213Z"
+                ts_tz_2: "2016-09-24T14:01:30.213+03:00"
+            }}
+        }"#,
+        )
+        .await?;
+
+        insta::assert_snapshot!(
+          run_query!(runner, r#"{ findManyParent { id child { date date_2 time time_2 time_tz time_tz_2 ts ts_2 ts_tz ts_tz_2 } } }"#),
+          @r###"{"data":{"findManyParent":[{"id":1,"child":{"date":"2016-09-24T00:00:00.000Z","date_2":"2016-09-23T00:00:00.000Z","time":"1970-01-01T13:02:20.321Z","time_2":"1970-01-01T10:02:20.321Z","time_tz":"1970-01-01T13:02:20.321Z","time_tz_2":"1970-01-01T10:02:20.321Z","ts":"2016-09-24T14:01:30.213Z","ts_2":"2016-09-24T11:01:30.213Z","ts_tz":"2016-09-24T14:01:30.213Z","ts_tz_2":"2016-09-24T11:01:30.213Z"}}]}}"###
+        );
+
+        Ok(())
+    }
+
+    async fn create_row(runner: &Runner, data: &str) -> TestResult<()> {
+        runner
+            .query(format!("mutation {{ createOneParent(data: {}) {{ id }} }}", data))
+            .await?
+            .assert_success();
+        Ok(())
+    }
+}
+
+#[test_suite(only(Postgres))]
+mod decimal {
+    fn schema_decimal() -> String {
+        let schema = indoc! {
+            r#"
+            model Parent {
+                #id(id, Int, @id)
+
+                childId Int? @unique
+                child Child? @relation(fields: [childId], references: [id])
+            }
+
+            model Child {
+              #id(id, Int, @id)
+
+              float    Float   @test.Real
+              dfloat   Float   @test.DoublePrecision
+              decFloat Decimal @test.Decimal(2, 1)
+              money    Decimal @test.Money
+
+              parent Parent?
+            }"#
+        };
+
+        schema.to_owned()
+    }
+
+    // "Postgres native decimal types" should "work"
+    #[connector_test(schema(schema_decimal))]
```

Comment:
:+1: This test should also run on driver adapters.

Reply:
It should be the case already, right? Since you made driver adapters _versions_ of the postgres connector test kit